api/src/Modules/staff/Controller/BaseStaff.php

```diff
@@ -155,6 +155,36 @@ public function __construct(Container $container)
     }
 
 
+    public static function install($container): void
+    {
+
+    }
+
+
+    public static function permissions($database): ?array
+    {
+        $result = ['api.delete.staff.all', 'api.get.staff', 'api.post.staff.all', 'api.put.staff.all',
+            'api.delete.staff.all'];
+
+        $values = Select::new($database)
+            ->columns('DepartmentID')
+            ->from('Departments')
+            ->orderBy('`DepartmentID` ASC')
+            ->fetchAll();
+
+        foreach ($values as $value) {
```

Comment:
Will this allow for the hierarchy where Division members can make changes to any department member within that division?

I'll admit that when I was working to add the RBAC to `ModuleStaff.php`, I more or less copied the logic that was happening in the ConCom RBAC to this for Divisions/Departments to make sure everything was the same as before.